contracts/modules/STO/USDTieredSTOFactory.sol

```diff
@@ -9,17 +10,17 @@ import "../../libraries/Util.sol";
  */
 contract USDTieredSTOFactory is ModuleFactory {
 
-    address public USDTieredSTOProxyAddress;
+    address public logicContract;
 
     /**
      * @notice Constructor
      * @param _polyAddress Address of the polytoken
      */
-    constructor (address _polyAddress, uint256 _setupCost, uint256 _usageCost, uint256 _subscriptionCost, address _proxyFactoryAddress) public
+    constructor (address _polyAddress, uint256 _setupCost, uint256 _usageCost, uint256 _subscriptionCost, address _logicContract) public
     ModuleFactory(_polyAddress, _setupCost, _usageCost, _subscriptionCost)
     {
-        require(_proxyFactoryAddress != address(0), "0x address is not allowed");
-        USDTieredSTOProxyAddress = _proxyFactoryAddress;
+        require(_logicContract != address(0), "0x address is not allowed");
+        logicContract = _logicContract;
         version = "1.0.0";
```

Comment:
Let's bump the version?

Reply:
I think we have the changes for USDTieredSTO in the 2.1.0 then yes we should bump the version